client/src/containers/example.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * This is where you will map the state / dispatch to props
+ * You will also need to import the component and action associated with the prop
+
+EXAMPLE
+
+import { connect } from 'react-redux';
+import VideoList from './../components/VideoList.js';    //import the component
+import changeVideo from './../actions/currentVideo.js';  //import the action
+
+const mapStateToProps = (storeData) => ({videos: storeData.videos});  //connects the prop to the state saved in the store
+const mapDispatchToProps = (dispatch) => ({                           //links the event handler to the store via dispatch
+  handleVideoListEntryTitleClick: (choice) => dispatch(changeVideo(choice))
+});
+
+const VideoListContainer = connect(mapStateToProps, mapDispatchToProps)(VideoList);
+
+****Will not have to map dispatch to components that do not use event handlers
+
+ */
```

Comment:
Example code and commented snippets should not be pushed to GitHub
to provide an example you should create a functioning model and comment it well

Reply:
Would like a reference to an industry standard or an explanation as to why it cant be on main branch temporarily as I think it adds value to have temporary guidance available. will Delete if there is no aid in it or if there is a solid reason why it should not be pushed.